arch/configure.defaults

```diff
@@ -16,7 +16,7 @@ CC              =       CONFIGURE_CC
 LD              =       $(FC)
 RWORDSIZE       =       CONFIGURE_RWORDSIZE
 PROMOTION       =       #-fdefault-real-8
-ARCH_LOCAL      =       -DNEC -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM
+ARCH_LOCAL      =       -DNEC -DNONSTANDARD_SYSTEM_SUBR CONFIGURE_D_CTSM
```

Comment:
I was wondering why -DWRF_USE_CLM was replaced. This is the flag to disable CLM compilation that we may still want. It might have been added because CLM took a long time to compile on some platforms.

Reply:
I guess the question is whether CTSM is intended to completely replace CLM.

Reply:
@dudhia 
Jimy,
The flag `CONFIGURE_D_CTSM` is now used for both. In the Config.pl script, if the LILAC/CTSM build is requested, then a flag for CTSM is supplied. If the standard build is requested, then the flag -DWRF_USE_CLM is supplied. The LILAC/CTSM and CLM schemes cannot co-exist due to namespace collision. This is a functional and correct implementation that allows both capabilities, but selected at compile-time.